app/services/solidus_bolt/webhooks/create_service.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module SolidusBolt
+  module Webhooks
+    class CreateService < SolidusBolt::BaseService
+      attr_reader :division_id, :event, :url
+
+      def initialize(event:, url:)
+        @event = event
+        @url = url
+        @division_id = SolidusBolt::BoltConfiguration.fetch.merchant_id
```

Comment:
merchant ID that we were using has now been renamed to division ID by Bolt.

Instead of creating a new field maybe we should rename the existing one

Reply:
You can check this at https://merchant-sandbox.bolt.com/developers where you will now find a division public id field as well

Reply:
ok, I agree that renaming it would be better.

Reply:
added a new commit to rename the field
check here [775bf8c](https://github.com/nebulab/solidus_bolt/commit/775bf8c62e330bca72bdc448be2cd28d4e8757e1)